src/Request/GetOfficesRequest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\EcontBundle\Request;
+
+class GetOfficesRequest extends Request
+{
+    private const ENDPOINT = 'NomenclaturesService.getOffices.json';
+    private const HTTP_METHOD = 'POST';
```

Comment:
I think it was `GET` but surprisingly it was using request's body to pass parameters. Is `POST` being used because of that?

Reply:
Exactly